pkg/controller/replicaset/replica_set.go

```diff
@@ -59,9 +58,8 @@ const (
 	statusUpdateRetries = 1
 )
 
-func getRSKind() schema.GroupVersionKind {
-	return v1beta1.SchemeGroupVersion.WithKind("ReplicaSet")
-}
+// controllerKind contains the schema.GroupVersionKind for this controller type.
+var controllerKind = v1beta1.SchemeGroupVersion.WithKind("ReplicaSet")
```

Comment:
It was written as a function to make the returned value "constant". Is there a reason to change it to a var?